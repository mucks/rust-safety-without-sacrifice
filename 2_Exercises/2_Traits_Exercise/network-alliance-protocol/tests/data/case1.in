# Basic model and report
STATE ForestCity 300
CITIZEN ForestCity 1 Alice Builder
CITIZEN ForestCity 2 Ben Researcher
CITIZEN ForestCity 3 Cara Validator
REPORT ForestCity
