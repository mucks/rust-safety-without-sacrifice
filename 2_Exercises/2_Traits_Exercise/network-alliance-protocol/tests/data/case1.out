REPORT ForestCity: pop=3 gdp=300 per_capita=100 allies=0
